recipes/ds_store/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ds_store" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ds_store-{{ version }}.tar.gz
+  sha256: e52478f258626600c1f53fc18c1ddcd8542fa0bca41d4bd81d57c04c87aabf24
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
```

Comment:
@marcelotrevisani I wonder if grayskull should add a comment with the warning:

```
scripts detected. Package not eligible for noarch.
```

Every time I see a recipe like this I have to check and, when it was built with grayskull it is correct but when it was't we are probably missing noarch.